x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -66,3 +76,85 @@ const HomePageComponent: React.FC<HomePageProps> = ({
 HomePageComponent.displayName = 'HomePage';
 
 export const HomePage = React.memo(HomePageComponent);
+
+const useInitSearchBarUrlParams = () => {
```

Comment:
Can we encapsulate this hook away from the HomePage component? maybe somewhere inside _x-pack/plugins/security_solution/public/common/hooks_?